configs/containers/docker-ubuntu-clang-mpich.yaml

```diff
@@ -235,8 +235,6 @@ spack:
         echo "export CXX=clang++" >> /root/.bashrc && \
         echo "export FC=gfortran" >> /root/.bashrc && \
         printf "[credential]\n    helper = cache --timeout=7200\n" >> /root/.gitconfig
-        # Source spack environment script for root
```

Comment:
Due to an upstream change in spack, these scripts are no longer needed/no longer exist.